proto/BUILD

```diff
@@ -0,0 +1,15 @@
+# Copyright 2019 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Intentionally left empty (for now).
```

Comment:
I think layers told us BUILD files don't need the licence :) So unless you insist on it, pls remove.

Reply:
I removed the copyright header from all files except `*.bzl`. I'd like to keep them in there (even if they are not required).